plugins/arcgis/web-app/projects/main/src/lib/arc-event/arc-event.component.scss

```diff
@@ -1,69 +1,16 @@
-.arc-layer-form {
+.arcEvent {
```

Comment:
The only css applied here is margin-top.  Display attribute is overridden by the parent.  Margin top with a similar bottom margin feels a bit out of balance.  I think we can remove this class all together and the reference in the html template.  If it feels a bit crowed between list items a good solution might be to add top and bottom margin to the mat-divider element.